### ISSUE TITLE
Making my own Notes Server the default Custom Sync Server?

### ISSUE THREAD

**Issue by deljones** [standardnotes/forum#2402]
**Describe the issue**
I have just recently installed Standard Notes and Standard Notes Server (Community Version). First of all many many thanks to all those involved in bringing this to us. It’s excellent and already it’s getting some heavy use by a couple of users including myself.

Everything is working well, with no problems at all, I just have a question if I may regarding the Custom Sync Server.

In order to get logged into the Notes server you have to enter your email address and password etc, then to connect to your own hosted Notes server you have to click Advanced Option, and check Custom Sync Server. The box is already filled in with

https://api.standardnotes.com/

The official site. Users then have to complete that box with the details of my Notes Server.

Is there a way that when you Click Advanced Option, Custom Sync Server, that the entry box could be pre-filled with my notes server instead? Its just another box that doesn’t have to be completed, plus, password managers will not be able to complete that unlike the users credentials.

In other words, it would be useful to cut that bit out completely if possible as people will only be connecting to my Notes Server.

If anyone can help with a solution or a “it just can’t be done”, then I’d be most grateful

Best wishes

dj

**Comment by Parkertg** [comment]
By the sounds of it you're still using the webapp or apps made by SN.  (and thus not self-hosting the web app - if you were doing that then I'm sure it wouldn't hard to adjust )

I do know that once you update the "server" value when using the webapp/desktop app.  that value stays stored in local storage for the next time you use app.  in the "-storage" entry you can find an entry for "server". 
![image](https://user-images.githubusercontent.com/46120246/212765449-f45da61f-0761-43ab-b4c8-df9b9f8727c9.png)

![image](https://user-images.githubusercontent.com/46120246/212765878-8fd2a145-a35a-4e50-84f3-de16e28a1acd.png)

perhaps theres something there that's helpful

**Comment by Parkertg** [comment]
Except I'm also pretty sure the app also has the "use custom server" checkbox disabled by default. so you'd perhaps need to still open that menu

**Comment by deljones** [comment]
Hi @Parkertg. Thanks for your help. Both the notes server and server app are self hosted. I tried the iOS app for notes but couldn’t upload files using the app so I went back to the SN web notes app which is self hosted. I was hoping I could go into a config file somewhere and doctor that. I’m going to have a look at your suggestion above, but it doesn’t matter if it’s the iOS app desktop app or website, I have to modify the custom sync server. Grrrr! 

Also when I use a web browser and quit it, I’ve set all my browsers to clear cookies and history, I’m wondering if that has anything to do with it. 

More experimenting i fear. Great software though. So far. 

Have a great day

dj

**Comment by Parkertg** [comment]
Ahh i that case i think it might be easy

From app repo readme:
> You can configure the DEFAULT_SYNC_SERVER environment variable to set the default server for login and registration.
> DEFAULT_SYNC_SERVER=https://sync.myserver

**Comment by deljones** [comment]
@Parkertg brilliant. Thanks very much indeed. Just what I’m looking for. 

Have a great day

dj
